src/pages/home/HeaderView.js

```diff
@@ -176,12 +178,31 @@ const HeaderView = (props) => {
                                     shouldUseFullTitle={isChatRoom || isPolicyExpenseChat || isThread || isTaskReport}
                                 />
                                 {(isChatRoom || isPolicyExpenseChat || isThread) && !_.isEmpty(subtitle) && (
-                                    <Text
-                                        style={[styles.sidebarLinkText, styles.optionAlternateText, styles.textLabelSupporting, styles.pre]}
-                                        numberOfLines={1}
-                                    >
-                                        {subtitle}
-                                    </Text>
+                                    <>
+                                        {isThread ? (
+                                            <PressableWithoutFeedback
+                                                onPress={() => {
+                                                    Navigation.navigate(ROUTES.getReportRoute(props.report.parentReportID));
+                                                }}
+                                                accessibilityLabel={subtitle}
+                                                accessibilityRole="link"
+                                            >
+                                                <Text
```

Comment:
When we added this component, the content is stretched and hence clicking the blank space was opening user chat instead of the chat details. This caused a regression [here](https://github.com/Expensify/App/issues/21215)